From: Fabien Spindler <fabien.spindler@inria.fr>
Date: Sun, 20 Jul 2014 15:36:45 +0900
Subject: CMakeLists.txt: install share data in visp-VERSION if necessary

* CMakeLists.txt: install share data packaged in libvispX.Y in
  visp-VERSION directory to avoid failures on updates.

Origin: upstream, http://bugs.debian.org/754657
Bug-Debian: http://bugs.debian.org/754657
---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e370fb9..b5c6fac 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -928,14 +928,14 @@ IF(USE_OGRE)
     if(UNIX)
       install(FILES
         ${VISP_BINARY_DIR}/install/resources.cfg
-        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/visp/data/ogre-simulator
+        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/visp-${VISP_VERSION}/data/ogre-simulator
         PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_WRITE
         COMPONENT libraries
       )
       if(OGRE_MEDIA_NOT_AVAILABLE)
         install(DIRECTORY
           data/ogre-simulator/media
-          DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/visp/data/ogre-simulator
+          DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/visp-${VISP_VERSION}/data/ogre-simulator
           FILE_PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_WRITE
           COMPONENT libraries
         )
@@ -2157,7 +2157,7 @@ IF(UNIX)
   foreach(SCENE ${WIREFRAME_SCENES})
     INSTALL(FILES 
       "${VISP_SOURCE_DIR}/src/simulator/wireframe-simulator/scene/${SCENE}"
-      DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/visp/data/wireframe-simulator
+      DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/visp-${VISP_VERSION}/data/wireframe-simulator
       PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_WRITE
       COMPONENT libraries
     )
@@ -2165,7 +2165,7 @@ IF(UNIX)
   foreach(SCENE ${ROBOT_ARMS_SCENES})
     INSTALL(FILES 
       "${VISP_SOURCE_DIR}/src/robot/simulator-robot/arms/${SCENE}"
-      DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/visp/data/robot-simulator
+      DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/visp-${VISP_VERSION}/data/robot-simulator
       PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_WRITE
       COMPONENT libraries
     )
